src/rgw/rgw_main.cc

```diff
@@ -594,12 +594,11 @@ static int process_request(RGWRados *store, RGWREST *rest, RGWRequest *req, RGWC
     goto done;
   }
 
-  if ( s->user.user_id != RGW_USER_ANON_ID ) {
-    if ( rgw_rate_limit_ok(s->user.user_id, op) == false ) {
-      dout(20) << "user: " << s->user.user_id << "exceeded API limit" << dendl;
-      abort_early(s, op, -ERR_SLOW_DOWN);
-      goto done;
-    }
+  /* Anonymous user is also regulated by rate limit, check the rate limit in config file.*/
```

Comment:
nitpick: maintain same coding style as rest of the file, ie. line formatting, comment style, etc -- see if your ide can help here